components/button/test/index.js

```diff
@@ -24,71 +24,67 @@ describe( 'Button', () => {
 			expect( button.type() ).to.equal( 'button' );
 		} );
 
-		it( 'with isPrimary', () => {
+		it( 'should render a button element with button-primary class', () => {
 			const button = shallow( <Button isPrimary /> );
 			expect( button.hasClass( 'button' ) ).to.be.true();
 			expect( button.hasClass( 'button-large' ) ).to.be.false();
 			expect( button.hasClass( 'button-primary' ) ).to.be.true();
 		} );
 
-		it( 'with isLarge', () => {
+		it( 'should render a button element with button-large class', () => {
 			const button = shallow( <Button isLarge /> );
 			expect( button.hasClass( 'button' ) ).to.be.true();
 			expect( button.hasClass( 'button-large' ) ).to.be.true();
 			expect( button.hasClass( 'button-primary' ) ).to.be.false();
 		} );
 
-		it( 'with isToggled', () => {
+		it( 'should render a button element with is-toggled without button class', () => {
 			const button = shallow( <Button isToggled /> );
 			expect( button.hasClass( 'button' ) ).to.be.false();
 			expect( button.hasClass( 'is-toggled' ) ).to.be.true();
 		} );
 
-		it( 'with disabled', () => {
+		it( 'should add a disabled prop to the button', () => {
 			const button = shallow( <Button disabled /> );
 			expect( button.prop( 'disabled' ) ).to.be.true();
 		} );
 
-		it( 'does not render target without href present', () => {
+		it( 'should not poss the prop target into the element', () => {
 			const button = shallow( <Button target="_blank" /> );
 			expect( button.prop( 'target' ) ).to.be.undefined();
 		} );
+
+		it( 'should render with an additional className', () => {
+			const button = shallow( <Button className="gutenberg" /> );
+
+			expect( button.hasClass( 'gutenberg' ) ).to.be.true();
+		} );
+
+		it( 'should render and additional WordPress prop of value awesome', () => {
+			const button = <Button WordPress="awesome" />;
+
+			expect( button.props.WordPress ).to.equal( 'awesome' );
```

Comment:
I don't think so.  I admit it is probably confusing.  This is just pure React, I don't believe Enzyme is touching this in anyway. I believe it would need to be wrapped in `shallow` for the API to be available.  I should probably just wrap it in shallow and do what you said.